changelogs/unreleased/80717.json

```diff
@@ -0,0 +1,5 @@
+{
+  "title": "Small screen: manage long title with option and fix scrollView in operation order details screen",
+  "type": "fix",
+  "packages": ["core", "manufacturing"]
+}
```

Comment:
This should be split in two changelogs as changes are not the same at all in the two packages:
- "Operation order: add scroll on details screen",
- "Header: cut long title on small screens"